### PR TITLE
Output improve/v20

### PR DESCRIPTION
**Pull request by victorjulien**
Output and error handling overhaul

Builds on #8224 and #7346, to create a simpler output scheme for the console.

suricata-verify-pr: 1043

### CODE REVIEW

**Comment by suricata-qa** [comment]
Information:
 

ERROR: QA failed on TREX_GENERIC_rule_time.

ERROR: QA failed on TREX_GENERIC_cfg_time.
 | field | baseline | test | % |
|--|--|--|--| 
| | TREX_GENERIC_stats_chk | | |
| .capture.kernel_drops | 0 | 254 | 0.00 | 

Pipeline 11243

File: src/util-debug.c [hunk 1]
```diff
@@ -53,13 +51,27 @@ SCEnumCharMap sc_log_level_map[ ] = {
...
+    { "Not set",        SC_LOG_NOTSET },
+    { "None",           SC_LOG_NONE },
+    { "E",              SC_LOG_ERROR },
+    { "W",              SC_LOG_WARNING },
+    { "i",              SC_LOG_NOTICE },
+    { "i",              SC_LOG_INFO },
+    { "i",              SC_LOG_PERF },
+    { "i",              SC_LOG_CONFIG },
+    { "d",              SC_LOG_DEBUG },
+    { NULL,             -1 }
```
Comment (jasonish) [inline src/util-debug.c:64]: Is there a convention for using caps for high priority errors?  I do find it a bit of an odd-looking inconsistency.

Comment (victorjulien) [reply src/util-debug.c:64]: It's essentially only using caps for error and warning, which is meant to draw attention.

**Comment by ct0br0** [comment]
Looks like the suricata.json output changed broke some QA parsing.

Was `This is Suricata` and now may be `Suricata x.x.x`. We'll have to adjust if this goes forward.

**Comment by victorjulien** [comment]
> Looks like the suricata.json output changed broke some QA parsing.
> 
> Was `This is Suricata` and looks to be `Suricata x.x.x`. We'll have to adjust if this goes forward.

Can you make that adjustment? Will merge those changes soon.

**Comment by ct0br0** [comment]
Should be just a syntax change. I'll run against this.

**Comment by ct0br0** [comment]
Actually, question. Is this going to change in 6 as well?

**Comment by victorjulien** [comment]
> Actually, question. Is this going to change in 6 as well?

No, just in master/7

**Comment by victorjulien** [comment]
replaced by #8302

**Pull request closed without merging**